elastic-tube-3d/solid-fenics/run.sh

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
 set -e -u
 
```

Comment:
```suggestion

. ../../tools/log.sh
exec > >(tee --append "$LOGFILE") 2>&1
```
Why is this part missing here?